tests/test_lib.py

```diff
@@ -839,13 +839,13 @@ def test_variable_components_many():
         return find_embedding(K, C, tries=1, chainlength_patience=0)
 
     @staticmethod
-    @unittest.skipIf(sys.platform == 'win32', 'Test may hang on win32, skipping out of caution')
+    @unittest.skipIf(os.getenv("CIRCLECI", None), 'Test is unreliable in CI')
```

Comment:
Didn't know that you could skip tests in CircleCI this way.

Reply:
Yeah, they have some [built-in environment variables](https://circleci.com/docs/variables/#built-in-environment-variables) for this purpose.

Reply:
Which kind of defeats the purpose of CI, but I feel the pain. I could never make neal/greedy/tabu/hybrid concurrency tests work reliably on AppVeyor. :disappointed:

Reply:
This test probably could/should be rewritten to be a bit more reliable. it relies on timing the interrupt. But I was trying to minimize changes here.